### ISSUE TITLE
Got error while install TextModels in Julia 1.6.1, MacOS Big Sur

### ISSUE THREAD

**Issue by tiendung** [JuliaText/TextModels.jl#27]
```
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.6.1 (2021-04-23)
 _/ |\__'_|_|_|\__'_|  |  Built by Homebrew (v1.6.1)
|__/                   |

(@v1.6) pkg> add TextModels
  Installing known registries into `~/.julia`
       Added registry `General` to `~/.julia/registries/General`
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package Flux [587475ba]:
 Flux [587475ba] log:
 ├─possible versions are: 0.4.1-0.12.4 or uninstalled
 ├─restricted by compatibility requirements with TextModels [77b9cbda] to versions: 0.9.0
 │ └─TextModels [77b9cbda] log:
 │   ├─possible versions are: 0.1.0 or uninstalled
 │   └─restricted to versions * by an explicit requirement, leaving only versions 0.1.0
 └─restricted by compatibility requirements with CuArrays [3a865a2d] to versions: [0.4.1-0.8.3, 0.11.0-0.12.4] or uninstalled — no versions left
   └─CuArrays [3a865a2d] log:
     ├─possible versions are: 0.2.1-2.2.2 or uninstalled
     └─restricted by julia compatibility requirements to versions: uninstalled
```

**Comment by iwasnothing** [comment]
I got same error

**Comment by samrose** [comment]
same error on mac os with julia 1.6.3

**Comment by p0ryae** [comment]
Since the repo owner/contributors haven't been active since forever, I recommend using my fork: https://github.com/p0ryae/TextModels.jl

Run `add https://github.com/p0ryae/TextModels.jl` in pkg mode, and you should be good.

TLDR: I only removed the specific version setting for each package, meaning every package will be added using their latest version. Although this sounds like it'll cause feature breaking in TextModels.jl, it all functions fine for my use case.
